lib/customer.rb

```diff
@@ -0,0 +1,41 @@
+require 'csv'
+
+class Customer
+  def initialize (id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
Do email and address need to be accessors?  Instead you should make them attr_reader methods.  This gives our classes client as limited opportunity as possible to change the instance variables incorrectly.